tendrl/gluster_integration/sds_sync/utilization.py

```diff
@@ -17,28 +21,39 @@ def sync_utilization_details(volumes):
                 }
             )
             continue
-        util_det = showVolumeUtilization(
-            volume.name
-        )
-        if util_det:
-            volume.usable_capacity = int(util_det['total'])
-            volume.used_capacity = int(util_det['used'])
-            volume.pcnt_used = str(util_det['pcnt_used'])
-            volume.total_inode_capacity = int(util_det['total_inode'])
-            volume.used_inode_capacity = int(util_det['used_inode'])
-            volume.pcnt_inode_used = str(util_det['pcnt_inode_used'])
+        volnames += volume.name + ','
+        voldict[volume.name] = volume
+    cmd = subprocess.Popen(
+        "tendrl-gluster-vol-utilization %s" % volnames,
+        shell=True,
```

Comment:
please comment before this subprocess call like to avoid memory leak we are doing CLI call. Otherwise again someone will change this logic in the future.

Reply:
Ack :)